package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@grammyjs/commands",
-  "version": "0.0.1",
+  "version": "0.2.0",
   "description": "grammY Commands Plugin",
   "main": "out/mod.js",
   "scripts": {
-    "prepare": "deno task backport"
+    "prepublishOnly": "deno task backport"
```

Comment:
Consider enabling npm installs from GH

Reply:
WDYM by that?

Reply:
It should be possible to run `npm i grammyjs/commands` and have the plugin from the main branch work locally. That works with all other libs, and `prepare` enables it.